wrappers/heapredirect.h

```diff
@@ -204,6 +212,38 @@ class HeapWrapper {
 
 } // namespace
 
+#if HL_USE_XXREALLOC
```

Comment:
Push the macro around xxrealloc to avoid duplication

Reply:
(as in, don't duplicate all the other boilerplate and just surround the one xxrealloc thing with `HL_USE_XXREALLOC`)